.rubocop.yml

```diff
@@ -24,3 +24,7 @@ Metrics/AbcSize:
 Metrics/BlockLength:
   Exclude:
     - lib/tasks/auto_annotate_models.rake
+
+Rails/SkipsModelValidations:
```

Comment:
I guess it's better to use inline disabling for migration files instead of polluting this file.